ext/fetch/23_response.js

```diff
@@ -157,6 +158,55 @@
     return resp;
   }
 
+  /**
+   * @param {Response} response
+   * @param {ResponseInit} init
+   * @param {{ body: __bootstrap.fetchBody.InnerBody, contentType: string | null } | null} bodyWithType
```

Comment:
Maybe link to the spec (as context for the numbered steps), something along these lines ?

```suggestion
   * @param {{ body: __bootstrap.fetchBody.InnerBody, contentType: string | null } | null} bodyWithType
   Implements https://fetch.spec.whatwg.org/#dom-response
```

Reply:
This is the algo it implements: https://whatpr.org/fetch/1392.html#initialize-a-response. Added a comment.